task/10-katas-1-tasks.js

```diff
@@ -137,7 +246,27 @@ function canDominoesMakeRow(dominoes) {
  * [ 1, 2, 4, 5]          => '1,2,4,5'
  */
 function extractRanges(nums) {
-    throw new Error('Not implemented');
+    let result = [],
+        count =0,
+        i = 0;
+        for (; i < nums.length; i++)
+        {
+            if ((nums[i + 1] - nums[i] == 1) && (nums[i + 2] - nums[i] == 2)) 
+            {  
+                count = 0;
+                while (nums[i + 1] - nums[i] == 1)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"